internal/uvm/create.go

```diff
@@ -196,6 +200,7 @@ func (uvm *UtilityVM) Close() (err error) {
 	windows.Close(uvm.vmmemProcess)
 
 	if uvm.hcsSystem != nil {
+		uvm.ReleaseCPUGroup(ctx)
```

Comment:
Probably be a good idea to log on error here. Also for terminate as well while we're at it

Reply:
Done.